elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -0,0 +1,390 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+import base64
+import os
+import json
+import re
+
+# Read https://prairielearn.readthedocs.io/en/latest/devElements/
+# Official documentation on making custom PL
+
+
+def render_html_colour(score):
+    # used to render the correct colour depending on student score
+    if score == 0:
+        return 'badge-danger'
+    elif score == 1.0:
+        return 'badge-success'
+    else:
+        return 'badge-warning'
+
+# courtesy of https://github.com/PrairieLearn/PrairieLearn/pull/2625
```

Comment:
its not really clear what this comment is referring to, or if it even needs to be here

Reply:
it's just acknowledging I copied those code from the original parsons branch lol, could be safely removed

Reply:
removed in latest commit